src/components/NewsModal/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {
+  useEffect, useState
+} from 'react';
+
+import {
+  Modal, Checkbox, CheckboxChangeEvent
+} from 'antd';
+
+import { useTranslation } from 'react-i18next';
+import ReactMarkdown from 'react-markdown';
+
+import rehypeExternalLinks from 'rehype-external-links';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+import remarkGfm from 'remark-gfm';
+
+import Logger from '@terrestris/base-util/dist/Logger';
+
+import { getBearerTokenHeader } from '@terrestris/shogun-util/dist/security/getBearerTokenHeader';
+
+import useAppSelector from '../../hooks/useAppSelector';
+import useSHOGunAPIClient from '../../hooks/useSHOGunAPIClient';
+
+import TextualContent from './TextualContents/model/TextualContent';
+import TextualContentService from './TextualContents/service/TextualContentService';
+
+export const NewsModal: React.FC = (): JSX.Element => {
+  const {
+    t
+  } = useTranslation();
+  const client = useSHOGunAPIClient();
+
+  const newsTextId = useAppSelector(state => state.newsText);
+  const newsTextPresent = newsTextId > -1 || false;
+  const [isModalOpen, setIsModalOpen] = useState(newsTextPresent && !JSON.parse(localStorage.getItem('hide-news-modal') || 'false'));
+  const [markdownContent, setMarkdownContent] = useState<string>('');
+  const [markdownTitle, setMarkdownTitle] = useState<string>('');
+
+  const handleCancel = () => {
+    setIsModalOpen(false);
+  };
+
+  useEffect(() => {
+    const fetchTextualContents = async () => {
+      const reqOpts = {
+        method: 'GET',
+        headers: {
+          ...getBearerTokenHeader(client?.getKeycloak())
+        }
+      };
+
+      try {
+        const textualContentService = new TextualContentService();
+        const textualContents = await textualContentService.findAll(undefined, reqOpts);
+
+        textualContents.content.forEach((textualContent: TextualContent) => {
+          textualContent.modified = new Date(textualContent.modified!);
+          textualContent.created = new Date(textualContent.created!);
```

Comment:
We should not force the presence of the field value if it is marked as optional to prevent null pointers.